.eslintrc

```diff
@@ -5,8 +5,7 @@
         "ol": false
     },
     "rules": {
-        "no-underscore-dangle": false,
-        "quotes": "single",
```

Comment:
This raises ``error  Strings must use doublequote`` when I run the testsuite - sure we want this?